<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Automata Celular 1D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>40</y>
      <width>262</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lengthLabel">
       <property name="text">
        <string>Longitud del Aut贸mata</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lengthLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>300</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ruleLabel">
       <property name="text">
        <string>Regla</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ruleLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>90</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Repeticiones</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="timeLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>300</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="initTapeGroupBox">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>11</y>
      <width>479</width>
      <height>149</height>
     </rect>
    </property>
    <property name="title">
     <string>Aut贸mata Inicial</string>
    </property>
    <widget class="QPushButton" name="randomPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Random</string>
     </property>
    </widget>
    <widget class="QLabel" name="percentLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>119</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Porcentaje 1's:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="percentLineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>70</width>
       <height>27</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tapeTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>45</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="evolutionGroupBox">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>171</y>
      <width>371</width>
      <height>389</height>
     </rect>
    </property>
    <property name="title">
     <string>Evoluci贸n del Aut贸mata</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="frequencyGroupBox">
    <property name="geometry">
     <rect>
      <x>408</x>
      <y>171</y>
      <width>370</width>
      <height>389</height>
     </rect>
    </property>
    <property name="title">
     <string>Diagrama de Frecuencia</string>
    </property>
    <widget class="QWidget" name="frequencyWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="runPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="action_Open_file"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_Open_file">
   <property name="text">
    <string>Abrir archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
